core/tests/webdriverio_utils/ExplorationEditorTranslationTab.js

```diff
@@ -183,7 +183,10 @@ var ExplorationEditorTranslationTab = function() {
     await action.click(
       'Confirm delete record button',
       confirmDeleteRecordButton);
-    await waitFor.pageToFullyLoad();
+    // We need to explicitly wait till the audio is deleted, in order
+    // to make sure there is no unsaved changes left in the exploration.
+    // eslint-disable-next-line oppia/e2e-practices
+    await browser.pause(2000);
```

Comment:
Can we not use some other alternative for this? Remember using browser.pause directly in the codebase is not a good idea in Oppia.

Reply:
Actually there is noting we can wait for when the audio is getting deleted (its a backend process). So, I was not able to come up with any other approach.

Reply:
Was there no toast messages present for this?

Reply:
No, we do not have any toast message present for this.